src/Plugin/Pluggable.php

```diff
@@ -380,9 +380,12 @@ public function fetchExtraPlugins(): array {
 	 * @throws Exception
 	 */
 	public function fetchPlugins(string $path = ''): array {
-		if ($path) {
-		} else {
-			$pluggable = $this;
+		// Todo @Don Hardman should fix this method. It produces error:
```

Comment:
Nice catch, it's fixed in the master now. This is a deprecated method. Feel free to rebase and fix it.